src/backend/launcher.ts

```diff
@@ -228,17 +230,27 @@ async function prepareLaunch(
     }
   }
 
+  if (
+    isUmuSupported(gameSettings.wineVersion.type) &&
+    !isInstalled('umu') &&
```

Comment:
This is missing an `await`, causing it to always be reported as `false` (Promises are truthy)
```suggestion
    !(await isInstalled('umu')) &&
```

What IDE are you using @Etaash-mathamsetty? Webstorm warns you about missing `await`s, pretty sure VSCode also does

Reply:
Vscode does not

Reply:
Vscode is probably more useless than a text editor (and eats up 30 gb disk space) so I might just switch to one of those

Reply:
Try setting up [no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises/) rule. IIRC it might catch that.

Reply:
Even if it did show the promise it's still not worth 30 gigs lol

Reply:
Correct. However, my point is that it's good to have automated checks catch issues like so, never mind what editor a contributor uses :wink: